lib/lightning_web/live/workflow_live.ex

```diff
@@ -15,6 +18,31 @@ defmodule LightningWeb.WorkflowLive do
     <Layout.page_content>
       <:header>
         <Layout.header title={@page_title} socket={@socket}>
+          <:details>
+            <%= case @live_action do %>
+              <% :index -> %>
+                <div>/ Workflows</div>
+              <% :new_job -> %>
+                <div>/ new workflow</div>
+              <% _ -> %>
```

Comment:
This introduces some strange UX, if you:

- Create a new workflow
- Set the name
- Click "Add your first Job"

This the workflow name changes from the one you entered to "new workflow", if you close the job panel it turns back.

What is the reason for changing the title here?